reana_client/cli/files.py

```diff
@@ -59,8 +57,16 @@ def files_group(ctx):
 @add_access_token_options
 @click.pass_context
 def get_files(ctx, workflow, _filter,
-              output_format, access_token):
-    """List workflow workspace files."""
+              output_format, access_token):  # noqa: D301
+    """List workspace files.
+
+    The `ls` command lists workspace files of a workflow which is set by
+    environment variable REANA_WORKON or provided as a command-line flag
+    `--workflow` or `-w`.
+
+    Example: \n
```

Comment:
You can always name the section "Examples:" because often there are several.

The example list can always start with the most simple use case.  Here, just `reana-client ls`.

Also note the technique using terms: myanalysis, mycode, mydata, myparameter, myvalue etc from REANA's main Getting Started guide.  We could repeat the same technique here.

Which would give:

```
Examples:
     $ export REANA_WORKON=myanalysis.42
     $ reana-client ls
     $ reana-client ls -w myanalysis.27
     $ reana-client ls -w myotheranalysis
```

Note that there is no need to keep extra blank line between examples... Provided the RTFD formatting is pretty?!